mobile/native/src/channel_trade_constraints.rs

```diff
@@ -45,14 +45,20 @@ pub fn channel_trade_constraints() -> Result<TradeConstraints> {
                 min_margin,
             }
         }
-        Some(channel) => TradeConstraints {
-            max_local_margin_sats: ln_dlc::get_usable_dlc_channel_balance()?.to_sat(),
-            max_counterparty_margin_sats: channel.counter_params.collateral,
-            coordinator_leverage,
-            min_quantity,
-            is_channel_balance: true,
-            min_margin,
-        },
+        Some(channel) => {
+            let total_collateral =
+                channel.own_params.collateral + channel.counter_params.collateral;
+            let local_balance = ln_dlc::get_usable_dlc_channel_balance()?.to_sat();
+            let counterparty_balance = total_collateral - local_balance;
```

Comment:
I like where this is going, but `local_balance` and `counterparty_balance` now mean different things:

- `local_balance` is the usable balance of the app, without the app trade margin (maybe including the fee reserve, I don't remember).
- `counterparty_balance` is the `counter_params.collateral`, plus the app trade margin and the coordinator trade margin (maybe including the fee coordinator reserve too).